electron/main/index.ts

```diff
@@ -265,6 +263,11 @@ async function createWindow() {
     return await baseCoin.recover(parameters);
   });
 
+  ipcMain.handle('broadcastTransaction', async (event, coin, parameters) => {
+    const baseCoin = sdk.coin(coin) as AbstractUtxoCoin;
+    return await baseCoin.broadcastTransaction(parameters);
+  });
+
```

Comment:
I know this is based off the other handlers, but ideally we should be:

1. Verifying `baseCoin` exists
2. Type guarding the `baseCoin` so the compiler can assign `AbstractUtxoCoin` instead of asserting it.

Reply:
the casting to AbstractUtxoCoin was not needed, it should throw if the coin doesnt support it or is an invalid coin anyways